uwrt_test_stress_can/CMakeLists.txt

```
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(uwrt_test_stress_can)
+
+#load catkin required dependencies
+find_package(catkin REQUIRED COMPONENTS
+        roscpp
+    )
+
+
+
+find_package(catkin REQUIRED COMPONENTS
+    roscpp
+)
+
+catkin_package(
+        INCLUDE_DIRS
+        include
+        lib/uwrt_mars_rover_hw_bridge/include
+        LIBRARIES
+        ${PROJECT_NAME}
+        uwrt-mars-rover-hw-bridge
+        CATKIN_DEPENDS
+        roscpp
+)
+
+
+# Specify additional locations of header files
+include_directories(
+        include
+        ${catkin_INCLUDE_DIRS}
+)
+
+# Declare and link the C++ Library
+add_library(${PROJECT_NAME}
+        src/uwrt_can.cpp
+        src/uwrt_can_test_node.cpp
+        )
```

you need an add_executable here for the main.cpp file you have something like this 
```suggestion
        add_executable(uwrt_test_stress_can src/main.cpp)
```